dashboard/src/deployment/DeploymentDetails.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { apiGet } from '../api/api.js';
+import { Accordion, Header, Icon, List, Segment } from 'semantic-ui-react';
+import Loading from '../util/Loading.js';
+//import CommandInstruction from '../util/CommandInstruction.js';
+
+const MemberGroupsView = ({member_groups}) => (
+  <div>
+    {member_groups.map((item) => <MemberListComponent 
+      group={item.group}
+      members={item.members}
+    />)}
+  </div>
+);
+
+const MemberListView = ({group, activeMemberID, onClick, members}) => (
+  <Segment>
+    <Header>{group}</Header>
+    <List divided>
+      {members.map((item) => <MemberView memberInfo={item} active={item.id === activeMemberID} onClick={onClick}/>)}
```

Comment:
May want to add `key={item.id}` to help React diffing renders when the members prop changes.